src/view/StrokeView.cpp

```diff
@@ -171,13 +174,15 @@ void StrokeView::drawWithPressuire()
 {
 	int count = 1;
 	double width = s->getWidth();
-	ArrayIterator<Point> points = s->pointIterator();
+    std::vector<Point>::const_iterator points = s->iteratorBegin();
```

Comment:
generally use the auto type for known types for return values, like:
`auto p_iter = s->iteratorBegin();`